src/Penrose/Functions.hs

```diff
@@ -1403,7 +1440,7 @@ at :: ConstrFn
 at [GPI o, Val (FloatV x), Val (FloatV y)] = (getX o - x) ^ 2 + (getY o - y) ^ 2
 
 lessThan :: ConstrFn
-lessThan [Val (FloatV x), Val (FloatV y)] = x - y
+lessThan [Val (FloatV x), Val (FloatV y)] = if x < y then 0 else (x - y)^2
```

Comment:
This change should be unnecessary since the output `x` of any constraint energy is composed with `max(x, 0)^2` which has the same effect... unless you saw that it actually improved the layout? (Which it shouldn't)

Reply:
Interestingly enough, it does! Here are examples from real analysis. My guess is that it's a unit issue. Since most other constraints are squared, I had to square this to make it "compatible" somehow?

`if` version:
![image](https://user-images.githubusercontent.com/11740102/68350135-6e79e780-00cd-11ea-88e0-b2656b7a0d38.png)
`x - y` version:
![image](https://user-images.githubusercontent.com/11740102/68350233-b436b000-00cd-11ea-9b21-d20558b27d5a.png)

Reply:
Wow, that's drastic. Good to know.